rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/adapter/CovidPatientVisitAdapterTest.java

```diff
@@ -0,0 +1,90 @@
+package io.ona.rdt.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.ona.rdt.PowerMockTest;
+import io.ona.rdt.R;
+import io.ona.rdt.domain.Visit;
+import io.ona.rdt.viewholder.CovidPatientVisitViewHolder;
+
+public class CovidPatientVisitAdapterTest extends PowerMockTest {
+
+    private final String visitName = "First Visit";
+    private final String visitDate = "12-15-2020";
+
+    private CovidPatientVisitAdapter adapter;
+    private List<Visit> list;
+
+    @Before
+    public void setUp() {
+
+        list = new ArrayList<>();
+        list.add(new Visit(visitName, visitDate));
+
+        adapter = new CovidPatientVisitAdapter(list, Mockito.mock(View.OnClickListener.class));
+    }
+
+    @Test
+    @PrepareForTest(LayoutInflater.class)
+    public void testOnCreateViewHolder() {
```

Comment:
Any reason not to extend RobolectricTest which would remove the need to mock the LayoutInflater?

Reply:
https://github.com/onaio/rdt-standard/pull/656#discussion_r551855904